src/YTable.cc

```diff
@@ -194,6 +194,7 @@ YTable::propertySet()
 	 * @property string	Item		QueryWidget only: Return one complete item
 	 * @property string	IconPath	Base path for icons
 	 * @property bool	MultiSelection	Flag: User can select multiple items (read-only)
+	 * @property map<ItemID> OpenItems 	Map of IDs of all open items (read-only)
```

Comment:
A map is parametrized by two types, key and value. We must mention both, and ideally a brief example.

This is not theoretical. I have no idea how to code a test for this without compiling all the PRs and trying it.

Reply:
It's been like that literally forever; that's also why I didn't do the only right thing: Use a list of IDs instead; because the map _value_ is completely bogus here, returning only one of "ID" or "Text". I kept that to retain backwards compatibility and consistency with the Tree widget. I am not sure who introduced that (`git blame` led me to the beginning of our git / svn history), but I can't imagine it was me.

So yes, we can make the mess prettier, but it will still remain a mess. The values of that map are useless.

Reply:
Added more docs for this.